datafusion/physical-plan/src/joins/utils.rs

```diff
@@ -920,71 +920,99 @@ pub(crate) fn append_right_indices(
     }
 }
 
-/// Get unmatched and deduplicated indices
+/// Get unmatched and deduplicated indices for specified range of indices
 pub(crate) fn get_anti_indices(
```

Comment:
maybe could you update this comment with an explanation of what `range` represents? Specifically, is it a range of indices in input_indicies to process?